global_planner/src/gradient_path.cpp

```diff
@@ -81,7 +81,8 @@ bool GradientPath::getPath(float* potential, double start_x, double start_y, dou
         // check if near goal
         double nx = stc % xs_ + dx, ny = stc / xs_ + dy;
 
-        if (fabs(nx - start_x) < .5 && fabs(ny - start_y) < .5) {
+        //if (fabs(nx - start_x) < .5 && fabs(ny - start_y) < .5) {
+        if (fabs(nx - start_x) < 1 && fabs(ny - start_y) < 1) {		//To be able to work with A* algorithm and GradientPath method
```

Comment:
Can you give an example of where the combination A*/Gradient was NOT finding a path that came within the distance before?

Reply:
When I run "teb_local_planner_tutorials" demo launch file which uses a hierarchical approach to plan a trajectory using A*/Gradient for global planning, I get these errors:
[ERROR]: NO PATH!
[ERROR]: Failed to get a plan from potential when a legal potential was found. This shouldn't happen.